docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,52 +1,198 @@
 ## BASIC01: The domain must have a parent domain
 
 ### Test case identifier
-**BASIC01** The domain must have a parent domain
+**BASIC01**
 
 ### Objective
 
-In order for a domain to work, it must be delegated to from a domain higher
-up in the DNS hierarchy (a parent domain). However, note that this
-requirement does not concern the root domain.
+In order for a domain (zone) to work, it must be delegated from a 
+zone higher up in the DNS hierarchy (a parent domain/zone). 
+This Test Case will determine if parent and child zones exist.
+
+If the zone to be tested is the root zone, it has no parent or
+delegation and will always pass this Test Case.
+
+If no parent can be determined, there cannot be any delegation.
+
+If the child zone does not exist (is not delegated), the only 
+test case to be run after this test case is BASIC03. However, 
+if the test type is an undelegated test, then all other test cases 
+can be run even if the child zone is not delegated.
+
+The Test Case must, in general, use the same algorithm as Method
+[Get-Parent-Zone].
 
```

Comment:
Right now the link does not work. It will work when this PR and the methods PR are merged into the same branch.

I found no good place for the text, but I agree that Objective is not the correct place. I have moved it.